application/_docker/database/scripts/01-ddl.sql

```diff
@@ -1,11 +1,6 @@
--- Drop tables if they exist, cascading dependencies
---DROP TABLE IF EXISTS "Membership" CASCADE;
---DROP TABLE IF EXISTS "MembershipType" CASCADE;
---DROP TABLE IF EXISTS "Account" CASCADE;
-
--- Create Account table
 CREATE TABLE "Account" (
     "accountId" BIGSERIAL PRIMARY KEY,
+    "mollieCustomerId" VARCHAR(255) NULL, -- TODO: Can an account exist without being registered as a customer at mollie?
```

Comment:
Volgensmij hadden we al besproken om mollie te gaan gebruiken, of heb ik dat fout?

Reply:
Het daadwerkelijke betaalsysteem zal een dummy zijn?

Reply:
**Welke bedoel je?**

Totaal niet aangesloten op een externe service.
_of_
Test modus binnen mollie (alle acties kunnen wel wordne uitgevoerd, alleen met nepgeld)


Ik zelf dacht aan het onderste namelijk

Reply:
Ik dacht ook eerder aan een dummy, hoewel het tweede mooi is, is het wel extra werk wat niet per se en must is

Reply:
Daar hebben julllie gelijk in. Dan zal ik het voor nu laten zoals het is. De requirement 
_"Als lid, wil ik mijn schuld kunnen betalen"_ betekent dus niet dat er een werkend betaalsysteem in moet zitten.